stories/rating-indicator/rating-indicator.stories.js

```diff
@@ -0,0 +1,433 @@
+                <label class="fd-rating-indicator__label" for="rating-max-value-7-3"></label>
+                
+                <input aria-label="4 star" type="radio" class="fd-rating-indicator__input" id="rating-max-value-7-4" name="rating-max-value-7" value=4">
+                <label class="fd-rating-indicator__label" for="rating-max-value-7-4"></label>
+                
+                <input aria-label="5 star" type="radio" class="fd-rating-indicator__input" id="rating-max-value-7-5" name="rating-max-value-7" value="5">
+                <label class="fd-rating-indicator__label" for="rating-max-value-7-5"></label>
+                
+                <input aria-label="6 star" type="radio" class="fd-rating-indicator__input" id="rating-max-value-7-6" name="rating-max-value-7" value="6">
+                <label class="fd-rating-indicator__label" for="rating-max-value-7-6"></label>
+                
+                <input aria-label="7 star" type="radio" class="fd-rating-indicator__input" id="rating-max-value-7-7" name="rating-max-value-7" value="7">
+                <label class="fd-rating-indicator__label" for="rating-max-value-7-7"></label>
+            </div>
+            <span class="dynamic-text">(2 of 7)</span>
+        </div>
+    </div>
+`;
+
+export const Rtl = () => `
+    <div class="example-container" dir="rtl">
+        <span>Rtl:</span>
+        <div class="fd-rating-indicator">
+            <div class="fd-rating-indicator__container" aria-label="Rating of this item is 2 out of 5">
+                <input aria-label="1 star" type="radio" class="fd-rating-indicator__input" id="rating-rtl-1" name="rating-rtl" value="1">  
+                <label class="fd-rating-indicator__label" for="rating-rtl-1"></label>
+                
+                <input aria-label="2 star" type="radio" class="fd-rating-indicator__input" id="rating-rtl-2" name="rating-rtl" value="2" checked>
+                <label class="fd-rating-indicator__label" for="rating-rtl-2"></label>
+                
+                <input aria-label="3 star" type="radio" class="fd-rating-indicator__input" id="rating-rtl-3" name="rating-rtl" value="3">
+                <label class="fd-rating-indicator__label" for="rating-rtl-3"></label>
+                
+                <input aria-label="4 star" type="radio" class="fd-rating-indicator__input" id="rating-rtl-4" name="rating-rtl" value=4">
+                <label class="fd-rating-indicator__label" for="rating-rtl-4"></label>
+                
+                <input aria-label="5 star" type="radio" class="fd-rating-indicator__input" id="rating-rtl-5" name="rating-rtl" value="5">
+                <label class="fd-rating-indicator__label" for="rating-rtl-5"></label>
+            </div>
+            <span class="dynamic-text">(2 of 5)</span>
+        </div>
+    </div>
+`;
+
+Rtl.parameters = {
+    docs: {
+        storyDescription: 'This rating indicator is displayed from right-to-left on the screen..'
```

Comment:
"The rating indicator can be displayed from right to left on the screen."

Also, there's something weird happening in the code example.. the numbers are jumbled, and the :Rtl is kind of awkward there. Anyway to put RTL on top, and right aligned?

Reply:
fixed